installer/cli/checkpoint.py

```diff
@@ -0,0 +1,84 @@
+import os
+import pickle
+from datetime import datetime
+from vm_providers.factory import get_provider_for
+
+
+CHECKPOINT_FILE = '.microk8s.checkpoint'
+CHECKPOINT_PATH = os.path.join(os.path.expanduser('~'), CHECKPOINT_FILE)
```

Comment:
A crazy idea. We could store some of this info inside the VM. Would this work better towards cross platform?